### ISSUE TITLE
[rllib] How to retrieve and modify the logits for PPO 

### ISSUE THREAD

**Issue by lucas-spangher** [ray-project/ray#15609]
I will repeat this question on your discussion page. 

I'd like to retrieve the logits that the PPO agent's policy computes for each multidiscrete action, and then modify them so that the action sampling can use my modified logits. 

I'm really sorry for the question, but the code is so abstracted that it's hard for me to tell how I would retrieve the logits and then reset them. Is it in the forward pass of ModelV2? Any direction you could provide would be helpful, thank you.

**Comment by sven1977** [comment]
Hey @lucas-spangher , thanks for the question! @mvindiola1 and I have answered it on discuss :)
https://discuss.ray.io/t/rllib-retrieve-and-modify-the-computed-discrete-action-logits-to-ppo-agent/2018
